components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -30,10 +32,10 @@ @implementation MDCTabBarView
 #pragma mark - Initialization
 
 - (instancetype)init {
-  self = [super init];
+  self = [super initWithFrame:CGRectZero];
   if (self) {
     _items = @[];
-    _itemViews = @[];
+    [self setUpSubviews];
```

Comment:
This looks far too complex to be called from `-init`.  Let's try to simplify and add the constraints later.

Reply:
Fixed.